config/config_defaults.go

```diff
@@ -1187,6 +1187,11 @@ const CGRATES_CFG_JSON = `
 "rankings":{								// RankingS config
```

Comment:
Please keep the right order in both config_defaults as well as in the config structures. The order should follow the one in trends.